### ISSUE TITLE
Can I modify a message sent ?

### ISSUE THREAD

**Issue by Mazen94** [matrix-org/matrix-js-sdk#1804]
This is my first time using matrix and I am stuck if there is a way to modify a message sent

**Comment by t3chguy** [comment]
Yes, you use `sendEvent` with specific content, see https://github.com/matrix-org/matrix-react-sdk/blob/68d194444a907b5dafcc88f7271a48cdb9310485/src/components/views/rooms/EditMessageComposer.tsx#L86-L109 as an example

**Comment by Mazen94** [comment]
@t3chguy  I can't find the content I need to send to edit the message :( if you have an example with this library
matrix-js-sdk

**Comment by t3chguy** [comment]
See the docs for the format of the event you send via `sendEvent` https://github.com/uhoreg/matrix-doc/blob/aggregations-edits/proposals/2676-message-editing.md#proposal

**Comment by Mazen94** [comment]
@t3chguy Thank you ! the message is modified but a new message is created with the old content
